examples/Header.main.tsx

```diff
@@ -12,6 +12,7 @@ import {
   IconButton,
   MenuItem,
 } from '@itwin/itwinui-react';
+import { SvgPlaceholder } from '@itwin/itwinui-icons-react';
 
 export default () => {
   const menuItems = (close: () => void) => [
```

Comment:
nit/opinion:

for the main example, can we remove `menuItems` and add a second `HeaderButton`? this way we get to more clearly showcase what `HeaderBreadcrumbs` does (it adds a chevron icon between two `HeaderButton`s)